public/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src 'self' https://meals-api-solanna.onrender.com">
```

Comment:
With this CSP directive, the inline styles  are prevented from being applied
` <div id="logon-register" style="display:none;">`

If you bring up developer tools in the browser, check the console and you can see the messages.

For testing purposes, I just removed the CSP and was able to test you ui.
But defining the CSP in this manner is a good security practice, but just requires that the inline styles get removed and added to a separate style sheet

Attached are modified versions of index.html and the separate style sheet file: index.css
[index.zip](https://github.com/AnnaSolovykh/jobs-api/files/13928496/index.zip)

Here's some more information:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src

Reply:
Oh I see... My bad, I am so sorry, I didn't open it on onRender assuming everything would be fines as it was fine locally... 

Thank you for the information and explanation!